pages/common/emulator.md

```diff
@@ -0,0 +1,36 @@
+# emulator
+
+> Manager Android emulators from the command line.
+> More information: <https://developer.android.com/studio/run/emulator-commandline>.
+
+- Display the help:
+
+`emulator -help`
+
+- Start an Android emulator device:
+
+`emulator -avd {{name}}`
+
+- Display the web cameras on your development computer that are available for emulation:
```

Comment:
Perhaps this will read better?
```suggestion
- Display the webcams on your development computer that are available for emulation:
```
Not sure that we need to expand `webcam` there.